docs_src/advanced_middleware/tutorial002.py

```diff
@@ -4,7 +4,9 @@
 app = FastAPI()
 
 app.add_middleware(
-    TrustedHostMiddleware, allowed_hosts=["example.com", "*.example.com"]
+    TrustedHostMiddleware,
+    allowed_hosts=["example.com", "*.example.com"],
+    www_redirect=True,
```

Comment:
```suggestion
    TrustedHostMiddleware, allowed_hosts=["example.com", "*.example.com"]
```
I think there is no need to pass default value here